torch/csrc/jit/passes/graph_fuser.h

```diff
@@ -19,6 +19,11 @@ constexpr size_t fusion_kernel_args_limit = 128;
 // On Windows will noop, NYI
 TORCH_API void FuseGraph(std::shared_ptr<Graph>& graph);
 
+TORCH_API void CustomFuseGraph(
+    std::shared_ptr<Graph>& graph,
+    std::function<bool(Node*)> fn,
```

Comment:
It would be better to call this `is_fusable` than `fn` because that name carries no information.